apps/src/studio/studio.js

```diff
@@ -787,7 +786,16 @@ var setSvgText = function (opts) {
       tspan.firstChild.data = tspan.firstChild.data.slice(0, len);
 
       if (opts.maxLines === tSpansAdded) {
-        return opts.fullHeight;
+        if (opts.width < opts.maxWidth) {
+          // Try again with a wider speech bubble
+          return setSvgText(Object.assign({}, opts, {
+            width: Math.min(opts.width * words.length / i, opts.maxWidth),
+          }));
```

Comment:
I take that back, doing it iteratively is fine, though it still adds and removes the tspans with every attempt.